config.proto

```diff
@@ -77,6 +77,9 @@ message DataCapture {
 
     // rpcBody enables/disables the capture of the request/response body in RPC
     Message rpcBody = 4;
+
+    // maximum value of span's attribute value in characters. Default should be 128.
+    google.protobuf.Int32Value attributeValueMaxLength = 5;
```

Comment:
Not sure if the nesting under `DataCapture` is the right way to go. Any suggestions are welcome.

Reply:
I think it belongs to `DataCapture` but not sure if it should be by itself. Maybe inside a `Span` message as we might add other attributes? I am not 100% either. Any idea @davexroth ?